package/samplers/auto_sampler/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .sampler import AutoSampler
```

Comment:
[nit] Currently, there are two ways to import `AutoSampler`. 

* `optunahub.load_module(package="samplers/auto_sampler").AutoSampler`
* `optunahub.load_module(package="samplers/auto_sampler").sampler.AutoSampler`

To avoid the latter import path, would it be better to rename the sampler module to `_sampler`?

```suggestion
from ._sampler import AutoSampler
```